honeybadgermpc/hbavss.py

```diff
@@ -308,13 +398,13 @@ def _get_dealer_msg(self, values):
                 z[k] = SymmetricCrypto.encrypt(
                     str(shared_key).encode("utf-8"),
                     (phi[k](i+1), aux_poly[k](i+1), witness))
-            dealer_msg_list[i] = dumps((commitments, ephemeral_public_key, z))
+            dispersal_msg_list[i] = dumps(z)
 
-        return dealer_msg_list
+        return dumps((commitments, ephemeral_public_key)), dispersal_msg_list
```

Comment:
Can we avoid a double `dumps` here? Can you not pickle them just here and get rid of the previous pickle?

Reply:
No, they are separate and one is for RBC and the other one (need to be a list) for AVID.